### PR TITLE
Update RDP Keyboard Languages

### PR DESCRIPTION
**Pull request by jeff-bb**
Using additional sources to add more languages / locales. Changed sort order to match other sections.

### CODE REVIEW

**Comment by timwoj** [comment]
Could you split up the sorting and the additions into two separate commits? As it is now, it's fairly difficult to review the changes.

**Comment by jeff-bb** [comment]
Apologies. I’ve made the change, hopefully that works (I’m a bit of a git noob).

**Comment by timwoj** [comment]
Yep, that looks much easier to review. Thanks 😃

File: scripts/base/protocols/rdp/consts.zeek [hunk 1]
```diff
@@ -97,42 +97,65 @@ export {
...
 		[2052] = "Chinese - People's Republic of China",
 		[4100] = "Chinese - Singapore",
 		[1028] = "Chinese - Taiwan",
-		[3076] = "Chinese - Hong Kong SAR",
-		[5124] = "Chinese - Macao SAR",
+		[31748] = "Chinese - Traditional",
 		[1155] = "Corsican",
-		[1050] = "Croatian",
 		[4122] = "Croatian (Bosnia/Herzegovina)",
+		[4122] = "Croatian (Latin) - Bosnia and Herzegovina",
```
Comment (timwoj) [inline scripts/base/protocols/rdp/consts.zeek:146]: This one looks duplicated from the one above it.

**Review by timwoj — CHANGES_REQUESTED** [review]
There's one duplicate in there that I only saw because it's right next to the same one. I double-checked the ones that got moved and they all look fine.

File: scripts/base/protocols/rdp/consts.zeek (hunk 1 above)
Comment (jeff-bb) [reply scripts/base/protocols/rdp/consts.zeek:146]: I blame Notepad as my IDE :)

**Review by jeff-bb — COMMENTED** [review]
I've removed the duplicate.

**Review by timwoj — APPROVED** [review]

**Pull request merged**